src/System.Runtime.Extensions/src/System.Runtime.Extensions.csproj

```diff
@@ -106,16 +143,42 @@
   </ItemGroup>
   <!-- WINDOWS: CoreCLR -->
   <ItemGroup Condition=" '$(TargetsWindows)' == 'true' And '$(CoreClrOrCorRt)' == 'true'">
+    <Compile Include="System\Environment.CoreCLR.cs" />
```

Comment:
This is not CoreCLR specific. The conventions used elsewhere are:
- .Windows.cs: Windows-specific implementation, usable for both classic and UWP apps
- .Win32.cs: Implementation specific for classic apps
- .WinRT.cs: Implementation specific for UWP apps